### PR TITLE
WebSocket inferencerunner v0

### PR DESCRIPTION
**Pull request by a10y**
Need to resolve a couple of issues before merge:

- [x] When updating anything else on the ModelPage, the InferenceRunner component does a full rerender from scratch
- [x] I'm hard-coding version `0.1.0`, should come from the form
- [x] Need to populate hyperparams from form
- [x] Need to populate real set of versions from the form

### CODE REVIEW

**Comment by tarrekshaban** [comment]
@a10y do you want to wait until the open items are done before we merge?

 Need to populate hyperparams from form
 Need to populate real set of versions from the form

File: frontend/src/api/core/request.ts
Comment (a10y) [inline frontend/src/api/core/request.ts:4]: This whole file is auto-generated shit from openapi cli

File: frontend/src/api/index.ts
Comment (a10y) [inline frontend/src/api/index.ts:18]: autogenerated from openapi cli

File: frontend/package.json
Comment (a10y) [inline frontend/package.json:11]: We don't use any of the generated service definitions so i figured we can just remove them from the repo for now

Comment (tarrekshaban) [reply frontend/package.json:11]: Makes sense to me. Good to remove bloat.

File: frontend/src/api/services/util.ts
Comment (tarrekshaban) [inline frontend/src/api/services/util.ts:1]: Nice! Thoughtful touch.

File: frontend/src/components/CompletionLivePreview.tsx
Comment (tarrekshaban) [inline frontend/src/components/CompletionLivePreview.tsx:1]: May be worth it to limit the size of the text that is generated before it si scrollable to prevent an ever-expanding text box for long responses.

Comment (tarrekshaban) [reply frontend/src/components/CompletionLivePreview.tsx:1]: Not blocking release fwiw just a thought

**Review by tarrekshaban — APPROVED** [review]
Thanks for the work on this, looks great. A couple good learnings here around Memoization in React.

**Review by tarrekshaban — APPROVED** [review]
Thanks for the work on this, looks great. A couple good learnings here (including Memoization in React)

**Review by tarrekshaban — COMMENTED** [review]
Thanks for the work on this, looks great. A couple good learnings here around Memoization in React.

**Pull request merged**
